redhat-cop/containers-quickstarts
# 🐛 ADD - YQ for yaml manip prior to helming stuff 🐛

#### What is this PR About?
I want to add `yq` for yaml manipulation to this slave. 
In my pipeline, I'm going to edit the helm char yaml before running a helm publish. ie set the app_version or the namespace for my chart etc  

#### How do we test this?
1. `docker build -t name/jenkins-slave-helm .`
2. `docker run --rm -it  --entrypoint=/bin/bash name/jenkins-slave-helm:latest`
3. Test: expected out put `test.passing == true`
```
cat << EOF | yq w - 'test.passing' true
---
test:
  passing: false
EOF
```

cc: @redhat-cop/day-in-the-life
